test/unit/operation_test.js

```diff
@@ -2478,6 +2478,20 @@ describe('Operation', function() {
         StellarBase.Operation.setTrustLineFlags({});
       }).to.throw();
     });
+    it('fails with a muxed account trustor', function() {
+      expect(() => {
+        const mAddress =
+          'MA7QYNF7SOWQ3GLR2BGMZEHXAVIRZA4KVWLTJJFC7MGXUA74P7UJUAAAAAAAAAAAAGZFQ';
+        console.log(mAddress);
+        StellarBase.Operation.setTrustLineFlags({
+          trustor: mAddress,
+          asset: StellarBase.Asset.native(),
+          flags: {
+            clawbackEnabled: false
+          }
+        });
+      }).to.throw();
+    });
```

Comment:
🤔  this test doesn't make sense to me. Do you need to test a trustline to the native asset? Won't it always fail due to it being the native asset?

Which kind of error are you expecting to throw here?

Reply:
This change shouldn't even be here; reverted!